verticapy/vdataframe.py

```diff
@@ -5993,9 +5993,7 @@ def load(self, offset: int = -1):
         """
         check_types([("offset", offset, [int, float],)])
         save = self._VERTICAPY_VARIABLES_["saving"][offset]
-        vdf = {}
-        exec(save, globals(), vdf)
-        vdf = vdf["vdf_save"]
+        vdf = pickle.loads(save)
```

Comment:
I think it would be better if you set the protocol of pickle. The default protocol for Python 3.6 and 3.7 is protocol 3, but it is protocol 4 for Python 3.8.
Protocol version 4 was added in Python 3.4 and is incompatible with previous versions. It adds support for very large objects, pickling more kinds of objects, and some data format optimizations.
I think you should set pickle to use protocol 4 for all supported Python versions.